### DISCUSSION TITLE
laser not functioning properly

### DISCUSSION THREAD

**Discussion by Clarky00** [arkypita/LaserGRBL#1009] [Q&A]
the laser isnt functioning properly. only had it 3 weeks . it worked at the beggining but now not fully engraving properly.

**Comment by Schwarzi07** [comment]
Please provide more Information.

**Comment by Clarky00** [comment]
Hi yes I will send images of what it’s doing . [Image.jpeg]

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by gmmanonymus111** [comment]
Sorry to tell you, but images sent by email don't appear on GitHub. Please, post them directly here.

**Comment by Clarky00** [comment]
[Image.jpeg]
Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Clarky00** [comment]
Has this worked ?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by gmmanonymus111** [comment]
`[Image.jpeg]` showing up, but there is no image :/

**Comment by Clarky00** [comment]
Ok I will try another way . Basically the laser only does half an image . It tried to do it but not consistent if that makes sense . ? If not could someone call me ?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by gmmanonymus111** [comment]
Without laser and/or engravery type, I'm guessing...
Does it say `Alarm` or `Pause` at the lower right corner when it stops?
![image](https://user-images.githubusercontent.com/9658243/82458406-eaaaaa00-9ab6-11ea-9969-570ba65ab361.png)
Or does it continue moving without the laser being powered on?

**Comment by Clarky00** [comment]
It’s dosnt stop .

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by gmmanonymus111** [comment]
Does it stop engraving at the same spot, or at randomly?
It might worth checking your TTL wiring, as it might come loose, rendering your laser 'always off'

**Comment by Clarky00** [comment]
Randomly .

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Clarky00** [comment]
Wiring checked.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Clarky00** [comment]
[Image.jpeg]This is what it’s doing . We’ve tried some today on some it’s fine then it will just start doing this ! .
Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by gmmanonymus111** [comment]
Images sent by email are not showing up.
![image](https://user-images.githubusercontent.com/9658243/82558329-b3e39b00-9b6d-11ea-8fdd-dfd6e3026c12.png)
Please paste them directly at the [issue](https://github.com/arkypita/LaserGRBL/issues/1009)

**Comment by Clarky00** [comment]
![thumbnail_IMG_7432](https://user-images.githubusercontent.com/65613033/82559600-d4a8f200-9b67-11ea-8cc8-19d51e1eed5d.jpg)
![thumbnail_IMG_7433](https://user-images.githubusercontent.com/65613033/82559625-dffc1d80-9b67-11ea-8813-f88cfd9b1e2e.jpg)

**Comment by gmmanonymus111** [comment]
What is your laser's optical power? It seems the laser fails to start lasing as the material is too light-coloured. Similar when you try to cut something from white paper, the laser sometimes does not have enough time to 'bite' in the paper, and can't start cutting.
Try lowering engraving speed or giving more power to your laser durnig your engraving job.
Or if you can't adjust those values, try darker wood.

**Comment by Clarky00** [comment]
Engraving speed 7800
S max to 585 .
We’ve printed on the same material for the last few days and it will be ok one min then not another .

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by StuartB4** [comment]
That's a fast engraving speed, Slow it down a bit.

**Comment by Clarky00** [comment]
![thumbnail_IMG_7435](https://user-images.githubusercontent.com/65613033/82575246-1ba2e180-9b80-11ea-9de0-6f1e1824aa54.jpg)
![thumbnail_IMG_7434](https://user-images.githubusercontent.com/65613033/82575250-1d6ca500-9b80-11ea-8e4e-c842c6445141.jpg)
We has changed the speed (faster and slower) we have also changed the s-max power up and down but nothing seems to be working and we have been using the same wood up to now with no problems.

**Comment by StuartB4** [comment]
It also looks like it might be slightly out of focus, have you tried adjusting the lens to get a smaller sharper dot. The laser dot is a rectangle not a round dot. So you really have to get it as small as possible. If you can see it with eye protection on then focus it you will notice that the sides of the rectangle start to move in to the center, if you go too far the top and bottom lines of the rectangle start to move apart.

**Comment by Karmael19** [comment]
HI,
We now have the same problem. It was a "small circle" and now it is this rectangle shape, I have tried to make it go back to the small circle without luck, any ideas?

**Comment by StuartB4** [comment]
When you focus it use a black non reflective surface, you can see the dot better. I'ts only a really small amount between in and out of focus.

**Comment by Karmael19** [comment]
Thank you for that. We ended up taking apart the laser completely and re-doing it. We found a few loose screws (lol not us mind you) and were able to re-focus the laser. If it happens again I will do what you suggested !

**Comment by StuartB4** [comment]
Ok. :+1:

**Comment by Clarky00** [comment]
Thank you we will have a look .

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
